fluentbit/td-agent-bit.conf

```diff
@@ -0,0 +1,56 @@
+[SERVICE]
+  Parsers_File parsers.conf
+
+# https://docs.fluentbit.io/manual/pipeline/inputs/systemd
+[INPUT]
+  Name systemd
+  Systemd_Filter _SYSTEMD_UNIT=atom-workshop.service
+  Strip_Underscores true
+
+# https://docs.fluentbit.io/manual/pipeline/filters/record-modifier
+# Drop all systemd fields - we only want the message
+[FILTER]
+  Name record_modifier
+  Match *
+  Allowlist_key MESSAGE
+
+# https://docs.fluentbit.io/manual/pipeline/filters/modify
+# Lowercase MESSAGE field for consistency
+[FILTER]
+  Name modify
+  Match *
+  Rename MESSAGE message
+
+# https://docs.fluentbit.io/manual/pipeline/filters/parser
+# Attempt to parse the message field, in case the app is logging structured data
+[FILTER]
+  Name parser
+  Match *
+  Key_Name message
+  Parser systemd_json
+
+# https://docs.fluentbit.io/manual/pipeline/filters/modify
+# Add app identity fields
+[FILTER]
+  Name modify
+  Match *
+  Add app atom-workshop
+  Add stage STACK_STAGE
```

Comment:
Is it worth adding a comment to say this value is replaced within the launch configuration?

Reply:
Good suggestion, I've done this.